### PR TITLE
Assistant Instructors

### PR DESCRIPTION
**Pull request by kurtisnelson**
No UI yet for adding an AI, it must be done in the console.

### CODE REVIEW

**Comment by kurtisnelson** [comment]
Fixes #97

File: app/models/cohort.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Cohort < ActiveRecord::Base
-  belongs_to :instructor
   belongs_to :campus
   has_many :students, dependent: :destroy
   has_many :assignments, dependent: :destroy
   has_many :days, dependent: :destroy
+  has_and_belongs_to_many :instructors
```
Comment (jah2488) [inline app/models/cohort.rb]: 😓

**Comment by jah2488** [comment]
Not a fan of `HABTM`, but that hardly seems like a reason to not give it a :+1:

**Comment by kurtisnelson** [comment]
Once we are satisfied with this PR in production, we need to remember to remove `instructor_id` from the cohorts table.

**Pull request merged**